judgments/context_processors.py

```diff
@@ -12,7 +14,11 @@ def cookie_consent(request):
 
     if cookie_policy:
         decoder = json.JSONDecoder()
-        decoded = decoder.decode(unquote(cookie_policy))
+        try:
+            decoded = decoder.decode(unquote(cookie_policy))
+        except json.JSONDecodeError:
+            raise SuspiciousOperation("Cookie tampered with: not valid JSON")
+            raise BadRequest("Cookie tampered with: not valid JSON")
```

Comment:
Does this work? Surely we'll never reach the `BadRequest` here because we've already raised `SuspiciousOperation`?

Reply:
No, the Suspicious Operation raises a Bad Request, that's gubbins.